lib/splash/having_trouble_screen.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:tutorial/constants.dart';
+import 'package:tutorial/splash/button.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class HavingTroubleScreen extends StatelessWidget {
+  const HavingTroubleScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        iconTheme: const IconThemeData(color: primaryColor),
+      ),
+      body: Center(
+        child: FractionallySizedBox(
+          widthFactor: 0.9,
+          heightFactor: 0.8,
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              const Spacer(flex: 5),
+              Image.asset('assets/havingtrouble.png'),
+              const Spacer(flex: 22),
```

Comment:
at this point we might want to use a set amount of space here vs the relative amount of using spacers